documentation/src/docs/metrics.md

```diff
@@ -115,25 +158,30 @@ public final class ServiceStarter {
 }
 ```
 
-## 2.1. Example for Dynamic Counter and OpStatsLogger(Timer)
-This is an example from io.pravega.segmentstore.server.host.handler.PravegaRequestProcessor. In this class, we registered two metrics: One timer (createStreamSegment), one dynamic counter (segmentReadBytes).
+## Example for Dynamic Counter and OpStatsLogger(Timer)
+
+This is an example from `io.pravega.segmentstore.server.host.handler.PravegaRequestProcessor`. In the class `PravegaRequestProcessor`, we have registered two metrics:
+
+- one Timer (`createStreamSegment`)
+- one dynamic counter (`segmentReadBytes`)
+
 ```java
 public class PravegaRequestProcessor extends FailingRequestProcessor implements RequestProcessor {
```

Comment:
Another copy-pasted code. 90% of this example is irrelevant to this document.

Reply:
Hyperlink provided